test/lib/image-type.test.ts

```diff
@@ -15,19 +15,20 @@ test("getImageType() returns the expected transports", (t) => {
     ImageType.DockerArchive,
     "docker-archive is handled",
   );
+  t.same(
+    getImageType("oci-archive:/tmp/nginx.tar"),
+    ImageType.OciArchive,
+    "oci-archive is handled",
+  );
 });
 
-test("getDockerArchivePath() returns the expected results", (t) => {
+test("getArchivePath() returns the expected results", (t) => {
   t.plan(2);
 
   t.same(
-    getDockerArchivePath("docker-archive:/tmp/nginx.tar"),
+    getArchivePath("docker-archive:/tmp/nginx.tar"),
     "/tmp/nginx.tar",
     "returns extracted path",
   );
-  t.same(
-    getDockerArchivePath("bad-path"),
-    "",
-    "does not handle errors and returns empty path",
-  );
+  t.throws(() => getArchivePath("bad-path"), "throws when provided bad path");
```

Comment:
Wondering if we should also validate the thrown error, e.g. `Error('The provided archive path is missing image specific prefix, eg."docker-archive:" or "oci-archive:"')`

Reply:
We can totally.